watsononlinestore/watson_online_store.py

```diff
@@ -74,6 +74,15 @@ def __init__(self, bot_id, slack_client,
         self.discovery_collection_id = os.environ.get(
             'DISCOVERY_COLLECTION_ID')
 
+        try:
+            self.discovery_score_filter = float(os.environ.get(
+                "DISCOVERY_SCORE_FILTER", 0))
+        except:
+            LOG.error("DISCOVERY_SCORE_FILTER must be a number, " +
```

Comment:
Don't use "except:".  In this case you should use except ValueError: because we know that's the one here and the error message is assuming that.  In general, I like "except Exception:" when I think I want to catch just about everything (and log it...).   I _think_ the way you are doing it is a little dangerous because it can swallow up interrupts and stuff. So I don't use that, but it is probably harmless here.